deploy/docker/Dockerfile-full

```diff
@@ -25,13 +25,29 @@ WORKDIR /nanomq/build
 
 RUN cmake -G Ninja -DENABLE_JWT=ON \
     -DNNG_ENABLE_TLS=ON \
+    -DNNG_ENABLE_QUIC=ON \
     -DNNG_ENABLE_SQLITE=ON \
     -DBUILD_ZMQ_GATEWAY=ON \
-    -DBUILD_BENCH=ON ..
+    -DBUILD_BENCH=ON .. && \
+    ninja
 
-RUN ninja install
+FROM ubuntu:22.04
```

Comment:
Looks the full image doesn't install zeromq, may be this is a problem
@JaylinYu

Reply:
thx

Reply:
line 41:42:  `COPY --from=builder ...`